extensions/nba-game-viewer/src/upcoming-games.tsx

```diff
@@ -1,7 +1,11 @@
+import { getPreferenceValues } from "@raycast/api";
 import Schedule from "./views/schedule";
 
 const Command = () => {
-  return <Schedule />;
+  const preferences = getPreferenceValues<{ league: string }>();
+  const selectedLeague = preferences.league || "NBA";
+
+  return <Schedule subtitle={selectedLeague} />;
```

Comment:
bug: the preference values are in lower-case. Also since we've defined a default value, `preferences.league` will not be undefined.

```suggestion
  const { league } = getPreferenceValues<Preferences>();

  return <Schedule subtitle={league.toUpperCase()} />;
```